packages/xo-web/src/xo-app/dashboard/overview/index.js

```diff
@@ -89,17 +89,22 @@ class PatchesCard extends Component {
 @injectIntl
 class DefaultCard extends Component {
   _getPoolWisePredicate = createSelector(
-    () => map(this.state.pools, 'id'),
-    poolsIds => item => isEmpty(poolsIds) || includes(poolsIds, item.$pool)
+    () => this.state.pools,
```

Comment:
The old behavior want to only observe the change of the pools' IDs because if a pool state change we shouldn't recompute the selector. 

This change will not respect the expected behavior and the current implementation also not respect it because `map` will always returns a new array, then, the selector will always be recomputed. 

The solution is to use `createCollectionWrapper`.